Exercises/strings/strings.js

```diff
@@ -0,0 +1,18 @@
+// STRINGS
+// 1. Assign string with unicode to a variable.
+let unicodeStr = 'Example of unicode symbol is \u00B9';
+console.log(unicodeStr);
+
+// 2. Declare a variable with string with escaped symbols.
+let escapeSymb = '\u00A5';
+console.log(escapeSymb);
+
+// 3. Declare a variable which will contain name of your favourite animal.
+// 4. Log a message My favourite animal is with dynamic variable from previous task.
+let favorCat = 'Murchyk';
+console.log(`My favourite animal is cat ${favorCat}`);
+
+// 5. Declare a variable which contain name and surname of your favourite writer in lowercase. Log his/her name with capitals first letters.
+let favorWriter = 'isaak asimov';
+let properFavorWriter = favorWriter[0].toUpperCase() + 'saak ' + favorWriter[6].toUpperCase() + 'simov';
+console.log(properFavorWriter);
```

Comment:
It's cheating) use slice/substr/substring instead of directly writing part of writer's name.

Reply:
By the way, you added unnecessary files to this pull-request, ideally only strings.js file should be present here.

Reply:
Solved by following code:

Reply:
Now any name + surname pair divided by gap will have first character in upper case

Reply:
About files, they were added in some weird way :) from another branch, so I deleted them in this branch now and, seems, should at first merge this PR and then merge other branch where this files exist to not loose the data.
I will try to find out how to avoid such behavior of git next time, maybe it's to not have several PRs with different branches in parallel, anyway it's strange for me :)